rocm-docs/caffe2-build.md

```diff
@@ -0,0 +1,89 @@
+# Caffe2: Building From Source on ROCm Platform
+
+## Intro
+This instruction provides a starting point to build caffe2 on AMD GPUs (Caffe2 ROCm port) from source.
+*Note*: it is recommended to start with a clean Ubuntu 16.04 system
+
+## Install ROCm
+
+Install ROCm stack following steps at [link](https://github.com/RadeonOpenCompute/ROCm/blob/master/README.md) if your machine doesn't have ROCm already.
+
+## Build caffe2 inside docker
+
+ If your machine doesn't have docker installed, follow the steps [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce) to install docker.
+
+### Pull the docker image
+```
+docker pull rocm/caffe2:unbuilt-rocm1.8.2
```

Comment:
Can we also provide a docker image which has Caffe2 built and installed on ROCm 1.8.2 for the user who merely wants to use C2? We can host it [here](https://hub.docker.com/r/rocm/)

Reply:
Yes. I will add the docker with caffe2 installed.